portfolio/src/main/webapp/script.js

```diff
@@ -28,3 +28,24 @@ function addRandomFact() {
   // Add it to the page.
   document.getElementById('random-facts-container').innerText = randomFact;
 }
+
+function displayMessages() {
+  fetch('/data').then(response => response.json()).then((messageArray) => {
+    
+    const messageList = document.getElementById('message-container');
+    messageList.innerHTML = '';
+
+    for (var i = 0; i < messageArray.length; i++) {
```

Comment:
you can change this to one line
`messageArray.forEach(msg => messageList.appendChild(createListElement(msg)));`